comp/metadata/inventoryagent/init_data.go

```diff
@@ -0,0 +1,122 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package inventoryagent
+
+import (
+	"context"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+	"github.com/DataDog/datadog-agent/pkg/util/hostname"
+	"github.com/DataDog/datadog-agent/pkg/util/installinfo"
+	"github.com/DataDog/datadog-agent/pkg/util/scrubber"
+	"github.com/DataDog/datadog-agent/pkg/version"
+)
+
+var (
+	// for testing
+	installinfoGet = installinfo.Get
+)
+
+func (ia *inventoryagent) initData() {
```

Comment:
Is there a specific reason to have this method not in the same file as other `inventoryagent` methods?

Reply:
This way we have the fetching/naming logic isolated. I can move it with the rest if you think it's better.